webapp/store/actions/appActions.js

```diff
@@ -21,6 +21,13 @@ export function getWindowInfo() {
   };
 }
 
+export function setClientsHydrated(hydrated = false) {
+  return {
+    type: 'CLIENTS_HYDRATED',
```

Comment:
We are defining the rest of the actions as constants right? Maybe we shouldn't just use the string literal here and instead import the constant

Reply:
Good catch. Did this while testing but should be a constant.

Reply:
Updated.